.travis.yml

```diff
@@ -1,16 +1,26 @@
 language: julia
+
 os:
     - linux
     - osx
+
 julia:
-    - 0.5
+    - 0.6
     - nightly
+
+matrix:
+    allow_failures:
```

Comment:
Because it isn't released and badges / PRs etc shouldn't be red because Base Julia decided to deprecate `+` or something.